pkg/utils/random/string.go

```diff
@@ -0,0 +1,24 @@
+package random
```

Comment:
We have a couple of random name generator in `pkg/ctl/cmdutils/namers.go` already.

Reply:
Thanks. Resolved this in latest version of this PR.